Server/Processors/ServerMessageProcessor.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Domain.Game;
+using Domain.Messages;
+using System;
+using System.Collections.Generic;
+
+namespace Server.Processors
+{
+    public class ServerMessageProcessor : IMessageProcessor
+    {
+        public List<Message> ProcessConnectedMessage(Message message)
+        {
+            var result = new List<Message>();
+
+            var connectionMessage = (ConnectionMessage)message;
+            Lobby.SetPlayerName(connectionMessage.PlayerId, connectionMessage.Name);
+
+            return result;
+        }
+
+        public List<Message> ProcessCreateGameMessage(Message message)
```

Comment:
Method expects CreateGameMessage type so it makes sense to change method signature to List<Message> ProcessCreateGameMessage(CreateGameMessage message) (in IMessageProcessor too)